source/javascripts/site.js

```diff
@@ -1,35 +1,58 @@
 // This is where it all goes :)
 
 var cardIndex = 0;
+var currentTopic = undefined;
 
 window.onload = function() {
-  var topicH1 = document.getElementById("card-topic-header");
+  var topicH1 = document.getElementById("card-title-header");
+  var topicDetails = document.getElementById("topics");
 
   var refreshButton = document.getElementById("refresh-button");
   var cardPrompt = document.getElementById("card-prompt");
   var card = document.getElementById("card");
-  
-  function drawCard() {
-    cardIndex = (cardIndex + 1) % cards.length;
-    cardPrompt.innerHTML = cards[cardIndex].prompt;
+  var cardBack = document.getElementById("card--back");
+
+  function drawCard(topic) {
+
+    if (topic) {
+      for(var i = 0; i < cards.length; i++){
+        cardIndex = (cardIndex + 7) % cards.length;
+        if (cards[cardIndex].topic == topic) {
+          break;
+        }
+      }
+    } else {
+      cardIndex = (cardIndex + 7) % cards.length;
+    }
 
+    cardPrompt.innerHTML = cards[cardIndex].prompt;
+    topicH1.innerHTML = cards[cardIndex].topic;
+    
     card.classList.add('card--transform');
+    cardBack.classList.add('cardBack--transform');
```

Comment:
You aren't using camel case anywhere else in the app, but are introducing it here. Thoughts on calling this something like `card-back--transform`?